### ISSUE TITLE
mov file recognized differently in browser and in server code

### ISSUE THREAD

**Issue by yoavain-sundaysky** [sindresorhus/file-type#211]
I'm using file-type in client code to determine content-type to use in request to server.
I'm using file-type in server code to make sure the content I get matches the content-type.
I used to get the same result, but today, after updating from 10.8.0 to 11.0.0 I got  2 different results:

Client:
```json
{
  "ext": "mp4",
  "mime": "video/mp4"
}
```

Server:
```json
{
  "ext": "mov",
  "mime": "video/quicktime"
}
```
Attached the file:
[640x360_vid.zip](https://github.com/sindresorhus/file-type/files/3173351/640x360_vid.zip)

**Comment by yoavain-sundaysky** [comment]
BTW, after reverting to 10.8.0 both server and client result is:
```json
{
  "ext": "mov",
  "mime": "video/quicktime"
}
```

**Comment by sindresorhus** [comment]
// @stroncium

**Comment by stroncium** [comment]
@sindresorhus I have no idea how to test it in browser, and I suppose it will be a waste of time to learn how is building for browsers done just for this issue, but while running the code in my mind I've come up with #214, which fixes an issue existing in nodejs, and there is really high chance it is the issue @yoavain-sundaysky hit in browser.

**Comment by yoavain-sundaysky** [comment]
Sorry for the late response.
I can confirm the problem is solved in 12.1
